src/lib/core/partialcubeop.ts

```diff
@@ -66,3 +66,17 @@ export function counterClockwiseQuarterTurn(
       }
     : { face, row, column };
 }
+
+export function halfTurn(
```

Comment:
for clarity's sake, what's a half vs a quarter turn defined as? half turn == 2 quarter turns? maybe add some comments documenting this 🤔

Reply:
Oh yeah, there's supposed to be an entire doc comment here but I forgot to put it in.